packages/lodestar-cli/src/cmds/beacon/cmds/init/index.ts

```diff
@@ -1,8 +1,9 @@
 import {CommandModule} from "yargs";
 import {IBeaconOptions} from "../../options";
 import {initHandler} from "./init";
+import {IGlobalArgs} from "../../../../options";
 
-export const init: CommandModule<IBeaconOptions, IBeaconOptions> = {
+export const init: CommandModule<IGlobalArgs, IBeaconOptions> = {
```

Comment:
Why is this type change necessary?

Reply:
Not sure, I'm trying to fix some type error but it would seem we are using these types completely wrong.
First one is passed in buidler commands and second in handler